.circleci/real_config.yml

```diff
@@ -4231,10 +4231,10 @@ workflows:
           name: test-e2e-slurm-misconfigured
           requires:
             - package-and-push-system-local-ee
-          filters:
-            branches:
-              only:
-                - main
+          # filters:
```

Comment:
im not gonna approve until comments are gone

Reply:
those are there to just make it easier for the reviewer to see the tests pass.

Reply:
I can just link to the CI run once it finishes

Reply:
maybe leave it in draft while you're testing. infra robot is pinging me for an automatic review even though the PR isn't quite ready